README.md

```diff
@@ -151,4 +150,4 @@ npm install
 
 ## Extras
 ### Add Susper as default Search Engine in Google Chrome
-![](https://media.giphy.com/media/JJW5Aj8LRIGbu/giphy.gif)
+![](https://media.giphy.com/media/JJW5Aj8LRIGbu/giphy.gif)
```

Comment:
What's the change here?

Reply:
@duttaditya18 please have a look at my comment from this PR:
https://github.com/fossasia/susper.com/pull/918
The last line from the file didn't has the \n character so I added it because it is recommended.

Reply:
All right got it!